builtin/mainmenu/dlg_version_info.lua

```diff
@@ -22,7 +22,7 @@ local function version_info_formspec(data)
 	return (
 		"size[9,4.5,true]" ..
 		"textarea[0.5,0.5;8.5,3;;" ..
-		fgettext("A new Minetest version was found!") .. ";" ..
+		fgettext("New Minetest version is available") .. ";" ..
```

Comment:
"A new Minetest version..." maybe? Sounds better and more like a full sentence... :)